third_party/move/move-model/src/model.rs

```diff
@@ -2003,6 +1992,78 @@ impl GlobalEnv {
         writer.extract_result()
     }
 
+    pub fn dump_fun(&self, fun: &FunctionEnv) -> String {
+        let tctx = &self.get_type_display_ctx();
+        let writer = CodeWriter::new(self.internal_loc());
+        self.dump_fun_internal(&writer, tctx, fun);
+        writer.extract_result()
+    }
+
+    fn dump_fun_internal(&self, writer: &CodeWriter, tctx: &TypeDisplayContext, fun: &FunctionEnv) {
```

Comment:
There is a dump for the entire env somehwere at the bottom of this file, I'm sure part of this code already exists there, can we unify this please to avoid duplication?

Reply:
This is actually the function part from `dump_env` extracted out as a routine to enable dumping just the function for debugging.